.github/workflows/test.yml

```
@@ -6,42 +6,35 @@ on:
       - master
 
 env:
-  CURRENT_STABLE_CHANNEL: nixpkgs-24.11-darwin
+  NIXPKGS_BRANCH: nixpkgs-unstable
+  NIX_DARWIN_BRANCH: master
+  NIX_VERSION: 2.24.11
 
 jobs:
-  test-stable:
+  test:
     runs-on: macos-13
     steps:
     - uses: actions/checkout@v4
-    - name: Install nix corresponding to latest stable channel
+    - name: Install Nix
       uses: cachix/install-nix-action@v30
       with:
-        install_url: https://releases.nixos.org/nix/nix-2.18.8/install
-    - run: nix flake check --override-input nixpkgs nixpkgs/${{ env.CURRENT_STABLE_CHANNEL }}
+        install_url: https://releases.nixos.org/nix/nix-${{ env.NIX_VERSION }}/install
+    - run: nix flake check --override-input nixpkgs nixpkgs/${{ env.NIXPKGS_BRANCH }}
 
-  test-unstable:
-    runs-on: macos-13
-    steps:
-    - uses: actions/checkout@v4
-    - name: Install nix from current unstable channel
-      uses: cachix/install-nix-action@v30
-      with:
-        install_url: https://releases.nixos.org/nix/nix-2.24.9/install
-    - run: nix flake check --override-input nixpkgs nixpkgs/nixpkgs-unstable
-
-  install-against-stable:
+  install:
```

```suggestion
  install-against-stable:
```

Until we can get LnL to change the required checks

---

Yep, done.